### DISCUSSION TITLE
TuringCourses/major/computer_vision/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#141] [General]
# TuringCourses/major/computer_vision/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/computer_vision/

**Comment by Yusux** [comment]
课程不算很难，如果仅仅是为了得分的话，为了最后的期末考试，留一天时间**对照提纲**（提纲可能不同年份有所不同）进行理解即可，考试时可以看到很多提前交卷的同学；相比之下，**潘老师班**的平时实验就需要下一定的功夫，尽量完成实验要求中所有内容就基本不会有问题，也可以获得上台展示的机会。

不过还是建议有时间的话认真听课，可以学到一些比较有意思的内容，比如 Transformer 和相机模型。老师讲的还是很透彻的。
